store/sqlstore/role_store.go

```diff
@@ -128,10 +130,11 @@ func (s *SqlRoleStore) Save(role *model.Role) (*model.Role, *model.AppError) {
 	return dbRole.ToModel(), nil
 }
 
-func (s *SqlRoleStore) createRole(role *model.Role, transaction *gorp.Transaction) (*model.Role, *model.AppError) {
+func (s *SqlRoleStore) createRole(role *model.Role, transaction *gorp.Transaction) (*model.Role, error) {
```

Comment:
It is not clear to me why we are migrating methods from role_store. I thought this PR was about scheme_store?

Reply:
Yep, but `SchemeStore` uses `RoleStore` when creating a new schema is required to create multiples roles, but the good part is that migrated method from `RoleStore` is internal and used mostly by `SchemeStore` but just once by itself, that is why I decided to migrate this method. You can see in the changes that their impact is only internal and the API of `RoleStore` was not changed :)

Reply:
Got it. Thanks for the clarification!